Tutorial.md

```diff
@@ -0,0 +1,121 @@
+# Tutorial on Show Design
+
+## Overview
+
+Sparkle Motion is a tool to design and run a light show.
+
+First, you define a "scene" -- an arrangement of physical light fixtures and their controller mappings.
+
+Next, you design a "show" -- a set of visual effects to project on a scene.
+
+Visual effects programmed as "shaders" -- small GLSL programs that process inputs like xy coordinates from the scene, colors from another shader, and audio metadata from the sound analysis plugin, and generate outputs like another xy coordinate for distortion or projection mapping, or a color for a light.
+
+Finally, you use the designer mode to design a UI for all the "controls" and run the light show in real time.
+
+## Simulator Quick Start
+
+On OS X, start the simulator:
+
+```
+brew install openjdk
+export PATH="/opt/homebrew/opt/openjdk/bin:$PATH"
+./gradlew --continuous jsRun
+```
+
+A brand new session starts with no Show or Scene loaded, and the Show/Scene menu open. For an existing session click the "☰" icon to open the Show/Scene menu.
+
+## Scene Quick Start
+
+Load a scene from the BAAAHS template. Defining a scene's model, light controllers and fixtures is an exercise for another day...
+
+### Load the BAAAHS template
+
+- "☰" to open the Show/Scene menu then Scene tab
+- "+ New Scene"
+- From template "BAAAHS" to see the sheep model 🐑
```

Comment:
I'd suggest using the "Hi-Res" template over BAAAHS. On the simulator, BAAAHS is pretty slow cuz the fake IP stack is (very) poorly optimized for lots of little packets. Hi Res is roughly the same number of pixels but way more responsive.

There's currently a sort-of bug in the template that makes Hi-Res look really dark which I'll fix. (I think the grid w/h just needs to be smaller; LEDs are rendered at a constant size in the visualizer and they're just too spread out right now.)

Reply:
Ok this definitely runs a lot faster! Thanks for the tip.